pkg/codegen/testing/test/testdata/components-pp/exampleComponent/main.pp

```diff
@@ -1,6 +1,10 @@
-config input string { }
+config input string {
+    description = "A simple input"
+}
 
-config cidrBlocks "map(string)" { }
+config cidrBlocks "map(string)" {
+    description = "The main CIDR blocks for the VPC"
```

Comment:
Lets make sure that one of these tests a multi-line string.

Reply:
Done ✅